### PR TITLE
Adds delay to patches

### PR DESCRIPTION
**Pull request by Supermichael777**
Three seconds at the moment. Its entirely reasonable in game.
:cl: Supermichael 777
rcsadd: There is now a slight delay to putting a patch on yourself.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1:

**Comment by Xhuis** [comment]
Gives a reason to using pills and balances instant 0 to 100 brute healing. :+1:

**Comment by Zilenan91** [comment]
no

**Comment by RandomMarine** [comment]
Bruise packs and ointment were given a self-delay for the same reason. :+1:

**Comment by Lumipharon** [comment]
A patch is like a bruisepack/ointment, only it instantly works, and applies to the entire body, and will heal ALL brute or burn on you.

**Comment by RandomMarine** [comment]
To play devil's advocate Lumi, patches don't come in stacks of six.

**Comment by Xhuis** [comment]
Pill bottles can fit up to seven patches.

**Comment by Lumipharon** [comment]
>Pill bottles can fit up to seven patches.

Exactly. I've asked goof why they even FIT in pillbottles, but I've never got an answer.
Also even if they didn't, 1 patch is more potent than 6 bruisepacks in most situations.

**Comment by RandomMarine** [comment]
One full 50u styptic patch does heal a whopping 300 brute (50 instant, 250 in reagent ticks). Although unlike bruise packs it can't be split among different people.

I'll agree that patches are superior except in the extreme unlikely case you have like six different people that all have 40 damage on a single limb. And even then you're taking a lot of time to scan, note which limb the damage is on, target limb, then finally click the wounded spaceman.

**Comment by Core0verload** [comment]
>I've asked goof why they even FIT in pillbottles, but I've never got an answer.

That's because patches are pill subtypes. Blame Goof.

**Comment by Shad0vvs** [comment]
This makes pills take 5 seconds to use on yourself instead of instantly. (I think its this one)

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	var/apply_type = INGEST
 	var/apply_method = "swallow"
 	var/roundstart = 0
+	var/self_delay = 0 //pills are instant, this is because patches inheret their aplication from pills
```
Comment (Xhuis) [inline code/modules/reagents/reagent_containers/pill.dm]: Wrong. Pills have no delay.

**Comment by Shad0vvs** [comment]
It seems like that yet people have been complaining.

**Comment by Xhuis** [comment]
Weird.

**Pull request merged**